### DISCUSSION TITLE
Items keep changing

### DISCUSSION THREAD

**Discussion by jlilly771** [PintaProject/Pinta#731] [Q&A]
I am trying to basically transcribe handwritten notes about maps for my work. The colors that I use are important. I will make text, make it a certain color and then click on the hand so it finalizes and then I will move onto the next item that may need to be a different color but when I change the colors for the new item I will find it will change the colors of other items that I have just finished.  I have tried saving in between different edits but I have to go back like 50 times to redo the same item over and over again. Am I doing something wrong? How do I get the items to stop changing?? The only thing I have found that kind of helps is to "save as" and then close the whole program and open it again but that is way to time consuming for the number of edits I need to do. And I say it kind of helps because I tried that at one point and it erased like 5 or 6 different items I had tried doing.  Any help would be appreciated!

**Comment by cameronwhite** [comment]
Could you file a bug for this please?
Are there cases where it changes the colours of multiple text items? I'd expect it might just change the last edited color
